telebot/__init__.py

```diff
@@ -5880,9 +5889,11 @@ def create_new_sticker_set(
                 raise ValueError('You must pass at least one sticker')
             stickers = [types.InputSticker(sticker=stickers, emoji_list=emojis, mask_position=mask_position)]
 
+        if sticker_format:
+            logger.warning('The parameter "sticker_format" is deprecated.')
```

Comment:
I'd clarify why: 
```suggestion
            logger.warning('The parameter "sticker_format" is deprecated since Bot API 7.2+. Stickers can now be mixed')
```